airbyte-integrations/connectors/destination-clickhouse-v2/src/main/kotlin/io/airbyte/integrations/destination/clickhouse_v2/spec/ClickhouseSpecification.kt

```diff
@@ -63,6 +64,14 @@ class ClickhouseSpecificationOss : ClickhouseSpecification() {
     @get:JsonProperty("password")
     @get:JsonSchemaInject(json = """{"order": 5, "airbyte_secret": true}""")
     override val password: String = ""
+
+    @get:JsonSchemaTitle("Enable JSON")
+    @get:JsonPropertyDescription(
+        "Use the JSON type when possible. If disabled, the JSON will be converted to a string."
```

Comment:
```suggestion
        "Use the JSON type for Object fields. If disabled, the JSON will be converted to a string."
```

(... and/or array/unions, see previous comment)